udfs/community/from_binary.sql

```diff
@@ -0,0 +1,30 @@
+  
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+-- from_binary:
+-- Input: STRING representing a number in binary form
+-- Output: INT64 number in decimal form
+CREATE OR REPLACE FUNCTION fn.from_binary(s STRING) AS 
+(
+  (
+    SELECT 
+      SUM(
+      	CAST(c AS INT64) << (LENGTH(s) - 1 - bit))
+    FROM 
+      UNNEST(SPLIT(s, '')) AS c WITH OFFSET bit
+  )
+);
```

Comment:
Add trailing newline

Reply:
Where to add a trailing newline? I don't see any issues with formatting.

Reply:
The file should end with a newline character. I believe GitHub is calling this out with the red symbol at the end of line 30.